UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Common/Logging/EventIds.cs

```diff
@@ -496,7 +496,23 @@ public enum EventIds
         /// <summary>
         /// 805122 - New access token is added to cache for external end point resource.
         /// </summary>
-        CachingExternalEndPointToken = 805122
+        CachingExternalEndPointToken = 805122,
+        /// <summary>
+        ///  805123 -Preparing to download ENC files based on Product/CellName EditionNumber and UpdateNumber from file share service.
+        /// </summary>
+        FileShareServicePreparingToDownloadENCFilesStart = 805123,
+        /// <summary>
+        ///  805124 -Completed download of ENC files based on Product/CellName EditionNumber and UpdateNumber from file share service.
+        /// </summary>
+        FileShareServicePreparingToDownloadENCFilesCompleted = 805124,
+        /// <summary>
+        /// 805125 - Preparing to query ENC files based on Product/CellName EditionNumber and UpdateNumber from file share service.
+        /// </summary>
+        FileShareServicePreparingToQueryENCFilesStart = 805125,
+        /// <summary>
+        /// 805126 - Completed query for ENC files based on Product/CellName EditionNumber and UpdateNumber from file share service.
+        /// </summary>
+        FileShareServicePreparingToQueryENCFilesCompleted = 805126
```

Comment:
Is it for preparation or actual query complete?

```suggestion
        FileShareServiceSearchQueryForSetOfENCsCompleted = 805126
```

Reply:
Yes updated EventId as per suggested, Thanks!